terraform/dotnet/k8s/deploy/resources/dotnet-frontend-service-depl.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dotnet-sample-app-deployment-${TESTING_ID}
+  namespace: dotnet-sample-app-namespace
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dotnet-sample-app
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: dotnet-sample-app
+      annotations:
+        instrumentation.opentelemetry.io/inject-dotnet: "true"
+    spec:
+      containers:
+        - name: dotnet-frontend
+          image: ${IMAGE}
+          ports:
+            - containerPort: 8080
+          env:
+            - name: LISTEN_ADDRESS
+              value: "0.0.0.0:8080"
+            - name: ASPNETCORE_URLS
+              value: "http://+:8080"
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: "http://cloudwatch-agent.amazon-cloudwatch:4316"
```

Comment:
Why do we need this here? It's not needed in any other EKS use case

Reply:
Removed, no longer needed because recent add-on update

Reply:
Removed